homeworks/rostyslav.havuka_RostyslavHavuka/12-httpclient/user-card/src/app/app.component.ts

```diff
@@ -0,0 +1,53 @@
+import { Component, OnInit } from '@angular/core';
+import { Subject } from 'rxjs';
+import { finalize } from 'rxjs/operators';
+import { User } from './models/user.model';
+import { UserService } from './services/user.service'
+import { LoaderService } from './services/loader.service';
+
+
+@Component({
+  selector: 'app-root',
+  templateUrl: './app.component.html',
+  styleUrls: ['./app.component.css']
+})
+
+export class AppComponent implements OnInit{
+  users: User[] = [];
+  items: Array<number> = [1, 2, 3];
```

Comment:
I have two questions here:
1) Why do you use `User[]` on the previous line and `Array<number>` on this? Use the one style. I suggest you use `number[]`;
2) Items are pages? So how do you know that you will have exactly 3 pages? What if the backend has more or less that 3 pages? Your pagination will be invalid 😄 . To solve this problem, get the numbers of pages from the backend response. It should return either the `total_count` and `items_per_page` fields (you can calculate pages by yourself) or the `pages` field